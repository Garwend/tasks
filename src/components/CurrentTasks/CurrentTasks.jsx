import Task from "../Task/Task";

const CurrentTasks = ({ currentTasks, setCompletedTasks, setCurrentTasks, setWaitingTasks }) => {
    return (
        <article>
            <header>Obecnie wykonywane zadania</header>
            <ul>
                {currentTasks.map(task => <Task key={task.id} {...task} setCompletedTasks={setCompletedTasks} setCurrentTasks={setCurrentTasks} setWaitingTasks={setWaitingTasks} />)}
            </ul>
        </article>
    )
}

export default CurrentTasks;