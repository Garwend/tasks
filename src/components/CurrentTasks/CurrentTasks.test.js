import renderer from 'react-test-renderer';
import CurrentTasks from './CurrentTasks';

test('should render CurrentTasks component', () => {
    const tasks = [{id:1, title:'test', priority: 3, seconds: 5}]
    const tree = renderer.create(<CurrentTasks currentTasks={tasks} />).toJSON();
    expect(tree).toMatchSnapshot();
})