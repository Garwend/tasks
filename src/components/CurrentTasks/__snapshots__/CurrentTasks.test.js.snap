// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CurrentTasks component 1`] = `
<article
  className="tasks-wrap"
>
  <header>
    Obecnie wykonywane zadania
  </header>
  <ul>
    <li
      className="task"
    >
      <p>
        <span>
          czas: 
        </span>
         
        5
         sekund 
      </p>
      <p>
        <span>
          priorytet: 
        </span>
         
        3
      </p>
      <button
        className="delete-btn"
        onClick={[Function]}
      >
        usu≈Ñ
      </button>
    </li>
  </ul>
</article>
`;
